src/utils/update.confirmation.statement.submission.ts

```diff
@@ -77,5 +78,11 @@ const generateSectionData = (section: SECTIONS, status: SectionStatus, extraData
         };
         return newRegisterLocationsData;
       }
+      case SECTIONS.TRADING_STATUS: {
+        const newTradingStatusData: TradingStatusData = {
```

Comment:
could you update the test for this module to cover lines 82 and 85? coverage is only just above the 80%